prow/cmd/deck/main.go

```diff
@@ -139,6 +141,7 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 	fs.StringVar(&o.templateFilesLocation, "template-files-location", "/template", "Path to the template files")
 	fs.StringVar(&o.gcsCredentialsFile, "gcs-credentials-file", "", "Path to the GCS credentials file")
 	fs.BoolVar(&o.rerunCreatesJob, "rerun-creates-job", false, "Change the re-run option in Deck to actually create the job. **WARNING:** Only use this with non-public deck instances, otherwise strangers can DOS your Prow instance")
+	fs.BoolVar(&o.acceptInsecureRequests, "accept-insecure-requests", false, "Allows insecure requests for CSRF and GitHub oauth.")
```

Comment:
I might name this just `--allow-insecure` or something instead, but I'm not terribly attached either way.

Reply:
Brevity... is the soul of wit (thanks autosuggest)